lesson_03/home_task_queries.sql

```diff
@@ -0,0 +1,87 @@
+/*
+ Завдання на SQL до лекції 03.
+ */
+
+
+/*
+1.
+Вивести кількість фільмів в кожній категорії.
+Результат відсортувати за спаданням.
+*/
+-- SQL code goes here...
+
+SELECT category.name, COUNT(1) as film_count
+FROM film
+JOIN film_category on film.film_id = film_category.film_id
+JOIN category on film_category.category_id = category.category_id
+GROUP BY category.name
+ORDER BY (2) DESC;
```

Comment:
Using numbers instead of field names is not recommended. I will explain why on a lesson

Reply:
BTW, no need to surround numbers with brackets